### PR TITLE
Add README + LICENSE

### PR DESCRIPTION
**Pull request by mjfaga**
Getting an initial README.md out there, as well as an MIT license.

Library usage examples coming soon!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+Craig Cavalier (@CraigCav)
+Mark Faga (@mjfaga)
+
+## Contributing
+
+[![CircleCI](https://circleci.com/gh/ezcater/lunar-core/tree/master.svg?style=svg)](https://circleci.com/gh/ezcater/lunar-core/tree/master)
+[![Maintainability](https://api.codeclimate.com/v1/badges/f7a71a45af56cfa68e24/maintainability)](https://codeclimate.com/github/ezcater/lunar-core/maintainability)
+[![Test Coverage](https://api.codeclimate.com/v1/badges/f7a71a45af56cfa68e24/test_coverage)](https://codeclimate.com/github/ezcater/lunar-core/test_coverage)
+
+### Getting Set Up
```
Comment (bauer-hillary-c) [inline README.md]: Maybe combine this with the `Installation` section above?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# Lunar Core
+Lunar Core provides a determinstic approach to querying the graphql-tools
```
Comment (bauer-hillary-c) [inline README.md]: typo: deterministic

File: README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+[mocking framework](https://www.apollographql.com/docs/graphql-tools/mocking.html)
+and getting back pre-defined object graphs.
+
+## Installation
+```
+npm i lunar-core lunar-express --save-dev
+```
+
+### Setup
+Lunar Core current has easy to use packages for use with:
```
Comment (tjwp) [inline README.md]: typo: currently

File: README.md (hunk 1 above)
Comment (mjfaga) [reply README.md]: This is a bit different from `Installation`. That covers getting Lunar up and running in an existing project to use as a testing framework. This section is a sub-section of `Contributing` and relates to getting set up to do just that :)

File: README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
+# Lunar Core
+Lunar Core provides a deterministic approach to querying the graphql-tools
```
Comment (CraigCav) [inline README.md]: How about  something like "provides the means to generate deterministic results from graphql-tools"?

I think it's worth emphasizing that it makes the mocked data deterministic, and therefore consistently repeatable (for example, for usage in tests).

**Pull request merged**